docs/user-event/api-convenience.mdx

```diff
@@ -55,6 +94,37 @@ hover(element: Element): Promise<void>
 pointer({target: element})
 ```
 
+```js
+const TestComponent = () => {
```

Comment:
Sorry I'm being a pain, but I think it would be better to revert this to the previous example.
The reason for this is to be framework/library-agnostic, this is now a react example.
I can't think of any other test for now 😅

The rest of the changes are looking good! Thanks for that.

Reply:
@timdeschryver 
That's fine, thanks for your review.
I reverted to the previous code. 🙏

Reply:
I think I was not being clear on what I meant.
The test for hover can be reverted to use `toHaveStyle`.

The rest of the tests were good how they were, but I see that `onChange` was added back (which was not the intention.

Reply:
@timdeschryver 
I'm sorry. I made a mistake. 🙏
I've modified the code again.

Reply:
@timdeschryver 
I understood you to remove the "hover/unhover" example code, is that correct?

Reply:
Not really, I can't find it the full snippet anymore because the commits are overwritten with the force push.
But I meant the initial example that was using `toHaveStyle`, or was this example always using a react component?

From our earlier conversation:

```
expect(hoverBox).toHaveStyle({backgroundColor: 'red'})
```

Reply:
```
expect(hoverBox).toHaveStyle({backgroundColor: 'red'})
```
@timdeschryver 
I apologize for the confusion caused by the force push.
This example also uses a React Component.

Reply:
@timdeschryver 
What about an example like this?
This is a test code for "hover/unhover" without using any React code.

```js
test('hover/unhover', async () => {
  const user = userEvent.setup()
  render(<div>hover</div>)

  const hoverBox = screen.getByText('hover')
  let isHover = false;

  hoverBox.addEventListener('mouseover', () => {
    isHover = true;
  })
  hoverBox.addEventListener('mouseout', () => {
    isHover = false;
  })

  await user.hover(hoverBox)

  expect(isHover).toBeTruthy()

  await user.unhover(hoverBox)

  expect(isHover).toBeFalsy()
})
```

Reply:
Oh my bad!
I'm fine with the second example using the event listener.
Thanks for following up!